dags/public-health/covid19/shelter-to-esri.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.operators.python_operator import PythonOperator
+from arcgis.gis import GIS
+
+# Shelter locations from Oscar 
+# on LASAN site 
+
+TIER_1_META_URL = "https://services1.arcgis.com/X1hcdGx5Fxqn4d0j/ArcGIS/rest/services/COVID_19_Shelters_Tier1/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=&returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token="
+
+TIER_2_META_URL = "https://services1.arcgis.com/X1hcdGx5Fxqn4d0j/ArcGIS/rest/services/COVID_19_Shelters_Tier2/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=&returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token="
+
+SHELTER_CSV_URL = "https://docs.google.com/spreadsheets/d/1pkg7PVCS4lwVhNA3TkMSWKmzw4vsd4W53_Q_Ou2mXqw/export?format=csv&id=1pkg7PVCS4lwVhNA3TkMSWKmzw4vsd4W53_Q_Ou2mXqw&gid=73455498"
+
+SHELTER_ID = "282aff8ca27447c8b4e7792671bc9dc3"
+def load_data(**kwargs):
+    """
+    Entry point for the DAG, loading shelter to ESRI.
+    """
+    gdf = gpd.read_file('./tier-1.geojson')
+    df = pd.read_csv(SHELTER_CSV_URL)
+    df = df.rename({'Shelter Site? ': 'ParksName'}, axis = 1)      
+
+    # need to string match join here, which is a pain. 
+
+    df = df.replace(to_replace='109th Recreation Center', value='109th Street Recreation Center')
+
+    # merge on parksname
+    # this preserves the number of entries (tested.)
+    gdf = gdf.merge(df, on='ParksName') 
+
+    # export to CSV
+    gdf['Latitude'] = gdf.geometry.x
+    gdf['Longitude'] = gdf.geometry.y
+    time_series_filename = "/tmp/shelter_timeseries_current.csv"
+
+    pd.DataFrame(gdf).to_csv(time_series_filename, index=False)
```

Comment:
What does this do to the `Point` column? Convert to WKT, or fail? We may want to drop it.

Reply:
adding commit to drop.